src/components/CodeBox.vue

```diff
@@ -33,7 +34,7 @@ function submit() {
   }
 }
 
-function clickedButton(word: IWord) {
+function selectBugWord(word: IWord) {
```

Comment:
What is the difference between `selectBugWord` and `selectBug`?
I can guess your intention with `selectBug` but not with `selectBugWord`...
Either find a clearer name, or document the method.
I'm fine with both.
The same is true for all `-BugWord` methods.
The concept of a bug word should be specified at some point.

Reply:
I think you mean `selectBugWord` and `submitBug`. 

`selectBugWord` selects a word as bug and `submitBug` submits the bug with  type and correction

I did this cause our data-structure uses words to handle the code and with the  -`Word` suffix I wanted to make clear that this  method uses the wordId. 
But I see why this leads to confusion and I will remove the suffix everywhere because the parameter also clarifies that this method is about a word with the id and the bug object also contains a word id.

I think the difference between `submit` and `select` is clear, so I think the code is self explaining after I remove the -`Word` suffix or should I add some Comments?